spec/puppetserver/ca/action/prune_spec.rb

```diff
@@ -95,13 +95,21 @@
   end
 
   describe 'update' do
-    it 'bumps CRL version up by 1' do
+    it 'bumps CRL number up by 1' do
       ca_key = OpenSSL::PKey::RSA.new(512)
       ca_cert = create_cert(ca_key, "Bazzup")
       ca_crl = create_crl(ca_cert, ca_key)
 
       subject.update_pruned_CRL([ca_crl], ca_key)
-      expect(ca_crl.version).to eq(2)
+
+      # crlNumber is based on the amount of certs in revoked list
+      # in our case, that number is 0. So crlNumber is 0.
+      ca_crl.extensions.each do |ext|
+        if ext.oid == "crlNumber"
+          ext.value=((ext.value.to_i + 1).to_s)
```

Comment:
I don't think you want to set this here. Shouldn't this already be getting updated to 1 when you call `updated_pruned_CRL` above?

Reply:
Oh darn copy and paste.